tests/test.rfb.js

```diff
@@ -2787,6 +2789,101 @@ describe('Remote Frame Buffer Protocol Client', function () {
                 expect(RFB.messages.pointerEvent).to.have.been.calledWith(
                     client._sock, 13, 9, 0x010);
             });
+
+            it('should send a single pointer event on mouse movement', function () {
+                client._handleMouseMove(100, 200);
```

Comment:
This is an internal function, so not really ideal test cases. Can we do things on a more proper level?

Reply:
I guess we could call `RFB._mouse.onmousemove`, but would that really be an improvement?

Reply:
Perhaps not. We really should change this code to use the modern event system.